### ISSUE TITLE
Support for fractions of a second

### ISSUE THREAD

**Issue by MobileMon** [rsrbk/SRCountdownTimer#21]
Feature request

Fractions of a second displayed instead of just int

For example, 1.2, 1.3, etc

I may try and work on this if I have time but not sure if author better understood it

**Comment by rsrbk** [comment]
Makes sense. I don't have time, but will appreciate you you'll work on it!
